src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -45,6 +77,23 @@ public SemanticsService() {
         eventTypes = new HashMap<>();
         eventTypes.put(ARTIFACT_PUBLISHED, EiffelArtifactPublishedEvent.class);
         eventTypes.put(ACTIVITY_FINISHED, EiffelActivityFinishedEvent.class);
+        eventTypes.put(ARTIFACT_CREATED, EiffelArtifactCreatedEvent.class);
+        eventTypes.put(ACTIVITY_CANCELED, EiffelActivityCanceledEvent.class);
+        eventTypes.put(ACTIVITY_STARTED, EiffelActivityStartedEvent.class);
+        eventTypes.put(ACTIVITY_TRIGGERED, EiffelActivityTriggeredEvent.class);
+        eventTypes.put(CONFIDENCELEVEL_MODIFIED, EiffelConfidenceLevelModifiedEvent.class);
+        eventTypes.put(ANNOUNCEMENT_PUBLISHED, EiffelAnnouncementPublishedEvent.class);
+        eventTypes.put(COMPOSITION_DEFINED, EiffelCompositionDefinedEvent.class);
+        eventTypes.put(CONFIGURATION_APPLIED, EiffelConfigurationAppliedEvent.class);
+        eventTypes.put(ENVIRONMENT_DEFINED, EiffelEnvironmentDefinedEvent.class);
+        eventTypes.put(FLOWCONTEXT_DEFINED, EiffelFlowContextDefinedEvent.class);
+        eventTypes.put(SOURCECHANGE_CREATED, EiffelSourceChangeCreatedEvent.class);
+        eventTypes.put(SOURCECHANGE_SUBMITTED, EiffelSourceChangeSubmittedEvent.class);
+        eventTypes.put(TESTCASE_FINISHED, EiffelTestCaseFinishedEvent.class);
+        eventTypes.put(TESTCASE_STARTED, EiffelTestCaseStartedEvent.class);
+        eventTypes.put(TESTSUITE_FINISHED, EiffelTestSuiteFinishedEvent.class);
+        eventTypes.put(TESTSUITE_STARTED, EiffelTestSuiteStartedEvent.class);
+
```

Comment:
create a method, probably static, that returns the HashMap and a static variable that you assign with the result from this method. This will allow us to use this service without need to instantiate SemanticsService.